configure.ac

```diff
@@ -1467,7 +1467,7 @@ if test "$with_hwloc_prefix" = "embedded" ; then
          CFLAGS="$HWLOC_EMBEDDED_VISIBILITY_CFLAGS $CFLAGS"
       fi
       hwlocsrcdir="src/hwloc"
-      hwloclib="src/hwloc/src/libhwloc_embedded.la"
+      hwloclib="src/hwloc/hwloc/libhwloc_embedded.la"
```

Comment:
Using `$HWLOC_EMBEDDED_LDADD` should make it so we don't have to manually discover the path.

Reply:
will fix this